maintenance/wikia/topTemplatesFromWiki.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ *
+ * @ingroup Maintenance
+ */
+
+require_once( dirname( __FILE__ ) . '../../Maintenance.php' );
+
+class allTemplatesFromWiki extends Maintenance {
+	const TEMPLATE_MESSAGE_PREFIX = 'articledi';
+	/** @var PipelineConnectionBase */
+	protected static $pipe;
+
+	/**
+	 * Set script options
+	 */
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = 'allTemplatesFromWiki';
+	}
+
+	public function execute() {
+		$data = $this->getTemplatesFromWiki();
+		$this->pushDataToRabbit( $data );
```

Comment:
Do you want to have output also before getting templates and informing about push start?